diagnostic.md

```diff
@@ -5,74 +5,83 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What is the purpose of a backend?
 
 ```md
-// your response here
+This is a place to store the data and be able to support multiple users
+accessing the app.
 ```
 
 Which layer in the MVC pattern is used by the controller to fetch data?
 
 ```md
-// your response here
+Model
 ```
 
 Which layer in the MVC pattern communicates with the model?
 
 ```md
-// your response here
-```
+Controller```
 
 Why don't we use views in our interpretation of the MVC pattern?
 
 ```md
-// your response here
+??
```

Comment:
We don't use views in our back-end because we can manage them on the front-end. This is the SPA way to do things, so we can minimize the amounts of requests we have to make to the server.